jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -293,11 +275,13 @@ public Resource addPath(String path) throws IOException
         {
             return resource;
         }
+
         if (resources != null)
         {
             return new ResourceCollection(resources.toArray(new Resource[0]));
         }
-        return null;
+
+        throw new MalformedURLException("path does not result in Resource: " + path);
     }
 
```

Comment:
The jetty maven plugin has a use-case for turning a ResourceCollection back into a csv list - any chance you could add that onto ResourceCollection?

Reply:
how would you handle a situation where the path itself has a comma?

Reply:
We need a proper CSV parser / generator in Jetty if this is going to work reliably. (we have the start of several of them sprinkled around the code, perhaps it's time to formalize it?)